hedera-node/src/main/java/com/hedera/services/ledger/HederaLedger.java

```diff
@@ -173,18 +181,6 @@ public void setTokenRelsLedger(
 		this.tokenRelsLedger = tokenRelsLedger;
 	}
 
-	public TransactionalLedger<AccountID, AccountProperty, MerkleAccount> getAccountsLedger() {
-		return accountsLedger;
-	}
-
-	public TransactionalLedger<NftId, NftProperty, MerkleUniqueToken> getNftsLedger() {
-		return nftsLedger;
-	}
-
-	public TransactionalLedger<Pair<AccountID, TokenID>, TokenRelProperty, MerkleTokenRelStatus> getTokenRelsLedger() {
-		return tokenRelsLedger;
-	}
-
 	/* -- TRANSACTIONAL SEMANTICS -- */
 	public void begin() {
```

Comment:
You need to add the `tokensLedger` to the `begin`, `rollback` and `commit` stages in the `TRANSACTIONAL SEMANTICS`